### DISCUSSION TITLE
Parse for Dice

### DISCUSSION THREAD

**Discussion by lolbat** [Obsidian-TTRPG-Community/fantasy-statblocks#66] [Q&A] [answered]
Does 'Parse for Dice' work with a table roller? And if so, how does one go about referencing it.

I have a table roller that references some other tables to create a mutation with random features.

`dice: [[Beastman Mutation^mutation]]|Mutation`

Is it possible to reference this inside the stat block so that I can include the mutation and have it formatted as per the style that I made for the Layout?

**Comment by FilBot3** [comment]
I've had spotty luck. Sometimes I just put the name of the block I added such as a `property line` that is named `spirit` and I "link" the dice to `spirit`. Some times it renders, other times, it doesn't.

**Comment by FilBot3** [comment]
@lolbat , you may want to re-post your question as a Q&A Post rather than discussion.

**Comment by lolbat** [comment]
I have. I actually find the emoji for Q&A a bit offensive and so I am loathe to use it.

**Comment by valentine195** [comment, marked as answer]
The plugin only supports dice that returns numbers.

Also, "Parse for dice" looks for a few specific strings in the text of the property by default, which are described [here](https://github.com/valentine195/obsidian-5e-statblocks#using-dice-rollers-in-layouts):

> The plugin will then attempt to parse the property content for common dice roll strings. The plugin will parse the following strings by default:
> 
> Rolling to hit (+15 to hit)
> Damage / healing (19 (2d10 + 8))
> A save (Strength +5)

Alternative you can use the dice callback, which allows you to define the dice string it returns.
